bramkouwen/Rock-Paper-Scissors-Game
# Improving Game Code

## What does this PR do?
* It Updates the game.py code to contain another function

## How do I check this PR?
* Pull down the branch 
* See the "quitoption" function 

## Screenshots 
None.
